runtime/src/legacy/mod.rs

```diff
@@ -100,21 +100,29 @@ where
 {
     type Service: Service;
 
-    async fn load(self, factory: Fac) -> Result<Self::Service, shuttle_service::Error>;
+    async fn load(
+        self,
+        factory: Fac,
+        logger: shuttle_service::Logger,
```

Comment:
Can we move `Logger` down to the runtime?

Reply:
Sure!